DSESwift/DSEConfig.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  DSEConfig.swift
+//  SensePlatform
+//
+//  Created by Tatsuya Kaneko on 01/12/15.
+//
+//
+
+import Foundation
+
+@objc public class DSEConfig: NSObject{
+    public var syncInterval: Double = 30 * 60// amount of seconds between upload sessions (default: 30 minutes)
```

Comment:
@kanekin for the sake of completeness, this is named uploadInterval in android. shouldn't we match the name for settings between android&iOS?